docker-compose.yml

```diff
@@ -813,6 +814,24 @@ services:
         source: .
         target: "C:/arrow"
     command: arrow\\ci\\scripts\\python_wheel_windows_test.bat
+  
+  python-wheel-manylinux-2014-java:
```

Comment:
We don't need to use the `python-wheel-manylinux-2014` prefix here despite that we depend on that image.

I think `java-<something>` would be more descriptive since this configuration is responsible for creating java jars with C++ libraries included.

Reply:
I will change the name for `java-bundled-jars` as you suggested in the first review